### DISCUSSION TITLE
Regarding one-time input in macros.

### DISCUSSION THREAD

**Discussion by burningcoffee1** [chhoumann/quickadd#515] [Ideas]
Firstly, I'd like to express my gratitude to the author for providing such a powerful plugin. Currently, I'm encountering a problem: when executing a macro that includes multiple Templates or Captures, I'm prompted with multiple input boxes. Most of the time, the input is the same, leading to repetitive input. Is there a way to input only once, and have each Template or Capture access the content inputted the first time?

**Comment by chhoumann** [comment]
> Firstly, I'd like to express my gratitude to the author for providing such a powerful plugin. 

Glad you like it :)

> Currently, I'm encountering a problem: when executing a macro that includes multiple Templates or Captures, I'm prompted with multiple input boxes. Most of the time, the input is the same, leading to repetitive input. Is there a way to input only once, and have each Template or Capture access the content inputted the first time?

Using e.g. `{{value:Title}}` will prompt you to enter a title. Any subsequent use of `{{value:Title}}` will simply be replaced by the value you first entered. This is true in Template & Captures, just as in Macros. Except, in Macros, the values propagate, such that any subsequent Template/Capture will also have access to it.

E.g. if you use a Template with `{{value:Title}}` in it, you can use `{{value:Title}}` later in your macro, and it'll reuse the value you entered.
